### ISSUE TITLE
UI: Change Font Size of Footer Links

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#223]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [x] User Interface
- [ ] Feature Request

## Description

We need to change the font size of the landing page footer links (Those under 'Support' &' Categories') to `14px`:

<p align="center">
  <img src="https://github.com/Sriparno08/Openpedia/assets/89148144/05900e88-c08f-497e-9c58-68a3d439d468">
</p>

**For Assignee:**

> Ensure that you submit a PR for this issue within 1 day. Otherwise, you will be unassigned from this issue and it will be passed on to the next contributor who asked to be assigned to it.

_Note: This issue is for JWoC 2024 participants. If you are not participating in this event, please refrain from asking to get assigned to this issue._

**Comment by Jenis07** [comment]
Can you please assign me this issue under JWOC?

**Comment by vishal2005025** [comment]
I have read the issue and I would like to work on this issue.
Please assign it to me under JWOC'2024.

**Comment by Sriparno08** [comment]
@Jenis07 You're assigned.

To Others:

The mentee who is assigned to this issue is chosen because he/she was the first one to ask for assignment with the 'JWOC' (Case-Sensitive) keyword in his/her comment.

@vishal2005025 You're already assigned to another issue. Once your PR for it is merged, you will be considered for other issues.

**Comment by Jenis07** [comment]
![image](https://github.com/Sriparno08/Openpedia/assets/100338684/57f3689a-bc03-4bd8-9b0e-c71b5469e485)
I have added the font size for this footer links..

**Comment by Sriparno08** [comment]
Looks good. You can make a PR now.
